Yoast/javascript
# Add Hebrew stemmer

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements Hebrew stemmer
* [Yoast SEO Premium] Introduces word form support for Hebrew

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

In Yoast SEO Premium:
* Change site language to Hebrew
* Add a Hebrew text of at least 300 words
* Set איחרנו as keyphrase and enter יאחר in the text/meta description/title. The relevant assessments should recognise יאחר as an instance of the keyphrase.
* Switch the keyphrase and the form in the text/meta description/title around and make sure the assessment results are still the same.
* Repeat the same process with ישלחו as keyphrase and ששלחה in the text/meta description/title.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-579
